website/content/docs/job-specification/service.mdx

```diff
@@ -200,6 +202,9 @@ service mesh][connect] integration.
     where no port mapping is necessary. This mode can only be set for services which
     are defined in a "group" block.
 
+  - `alloc_ipv6` - Identical as `alloc` but will pick the IPv6 address in case of
+    dual-stack or IPv6 only.
```

Comment:
```suggestion
  - `alloc_ipv6` - Same as `alloc` but use the IPv6 address in case of dual-stack or IPv6-only.
```

Updating to use the same language as L153